<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>日期时间</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>读取当前日期时间</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>字符串显示</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTimeEdit" name="timeEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>qDebug-Time</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>设置时间fromString</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>qDebug()--Date</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>设置日期fromString</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>qDebug()--DateTime</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>日期时间fromString</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>日历选择</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>选择的日期</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCalendarWidget" name="calendarWidget"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
